### PR TITLE
HartId/ResetVector: Correct the input width calculation

### PR DESCRIPTION
**Pull request by mwachs5**
**Related issue**: Follow up to #2531 which introduced this bug (https://github.com/chipsalliance/rocket-chip/blame/master/src/main/scala/tile/Core.scala#L134)

**Type of change**: bug report

**Impact**:  API modification

**Development Phase**: implementation

**Release Notes**

Fix a bug in the computation of HartID and ResetVector width going into the Tile due to improper use of Chisel compatibility mode functions.

### CODE REVIEW

File: src/main/scala/tile/Core.scala [hunk 1]
```diff
@@ -3,7 +3,7 @@
 package freechips.rocketchip.tile
 
 import Chisel._
-
+import Chisel.ImplicitConversions._
```
Comment (jackkoenig) [inline src/main/scala/tile/Core.scala:6]: I don't actually understand how this provides `.W`

Comment (jackkoenig) [reply src/main/scala/tile/Core.scala:6]: Via indirection, this is just https://github.com/freechipsproject/chisel3/blob/387274784115bee2bf7167547a7ee459219e1413/src/main/scala/chisel3/util/ImplicitConversions.scala#L12 which provides implicit conversion from `Int` to `UInt` and `Boolean` to `Bool`.

Comment (mwachs5) [reply src/main/scala/tile/Core.scala:6]: ok, so I should just get rid of this import?

Comment (mwachs5) [inline src/main/scala/tile/Core.scala:6]:
```suggestion
```

**Pull request merged**